lib/maths/time_series/CTimeSeriesTestForChange.cc

```diff
@@ -649,17 +704,14 @@ CTimeSeriesTestForChange::timeShift(double varianceH0,
 
     auto predictor = this->bucketPredictor();
 
-    // If the variance of the values is small compared to the predictor on the
-    // test interval we can't trust the shifted residual variance estimates.
-    TMeanVarAccumulator moments[2];
+    TMeanVarAccumulator valueMoments;
```

Comment:
Did the comment became obsolete? Or can we update the formulation to keep the logic explained?